hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -52,6 +52,9 @@
   private HddsProtos.ReplicationType type;
   private HddsProtos.ReplicationFactor factor;
   private FileEncryptionInfo encInfo;
+  private long objectID;
+  private long updateID;
+
```

Comment:
Also what happens for deployed clusters where the persisted keys have no object Id or update ID? What do we initialize it to?

Reply:
It will default to 0 as we use long primitive type here.